modules/apps/captcha/captcha-taglib/src/main/java/com/liferay/captcha/taglib/servlet/taglib/CaptchaTag.java

```diff
@@ -70,6 +75,22 @@ private String _getURL(HttpServletRequest httpServletRequest) {
 			return _url;
 		}
 
+		try {
+			CaptchaConfiguration captchaConfiguration =
+				ConfigurationProviderUtil.getSystemConfiguration(
+					CaptchaConfiguration.class);
+
+			if (captchaConfiguration.enableSimpleCaptchaHeadlessAPI()) {
+				return "/o/captcha/v1.0/simple?portletId=" +
+					PortalUtil.getPortletId(httpServletRequest);
```

Comment:
Could move the code in line 100 above `CaptchaConfiguration captchaConfiguration = `
and use the variable here.

Reply:
Fixed it. Thanks